### PR TITLE
Added cookbook for blocking users

### PR DESCRIPTION
**Pull request by martinmitrevski**
### 🔗 Issue Links

Resolves https://stream-io.atlassian.net/browse/PBE-4283?atlOrigin=eyJpIjoiNTdhMmEwNWRiZWNlNGUwOTkzZjQ2NmEwZjJkZTE2MjgiLCJwIjoiaiJ9.
 
### 🎯 Goal

Add a cookbook showcasing the block users feature.

### 📝 Summary

Doc updates.

### 🛠 Implementation

Just docusaurus updates, the implementation is in the demo app for SwiftUI.

### 🎨 Showcase

_Add relevant screenshots and/or videos/gifs to easily see what this PR changes, if applicable._

| Before | After |
| ------ | ----- |
|  img   |  img  |

### 🧪 Manual Testing Notes

_Explain how this change can be tested manually, if applicable._

### ☑️ Contributor Checklist

- [ ] I have signed the [Stream CLA](https://docs.google.com/forms/d/e/1FAIpQLScFKsKkAJI7mhCr7K9rEIOpqIDThrWxuvxnwUq2XkHyG154vQ/viewform) (required)
- [ ] This change should be manually QAed
- [ ] Changelog is updated with client-facing changes
- [ ] Changelog is updated with new localization keys
- [ ] New code is covered by unit tests
- [ ] Comparison screenshots added for visual changes
- [x] Affected documentation updated (docusaurus, tutorial, CMS)

### 🎁 Meme

Alt Text

### CODE REVIEW

File: docusaurus/docs/iOS/swiftui/swiftui-cookbook/blocking-users.md [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+
+## Introduction
+
+Blocking users is an essential feature in a chat app because it enhances user safety and experience. It allows individuals to protect themselves from harassment, spam, and unwanted interactions. By giving users control over their interactions, it helps maintain privacy, reduces the risk of cyberbullying, and promotes a respectful community atmosphere. 
+
+As a result, some app stores require this functionality as part of their review process.
+
+## Stream Chat
+
+The Stream Chat SDK provides blocking and unblocking users out of the box, as well as listing them.
```
Comment (laevandus) [inline docusaurus/docs/iOS/swiftui/swiftui-cookbook/blocking-users.md:14]: I was thinking about the "out of the box" term.

Not sure if something like this is better.
`The Stream Chat SDK provides a way for blocking and unblocking users, as well as listing all of the blocked users.`

File: docusaurus/docs/iOS/swiftui/swiftui-cookbook/blocking-users.md [hunk 2]
```diff
@@ -0,0 +1,197 @@
...
+
+When you block a user, you won’t receive any direct messages from that user anymore. However, if you share a group with other participants, you will still receive messages from the blocked user.
+
+In this cookbook, we will see how to implement this feature in your chat apps, using the Stream Chat SDK.
+
+## Low-Level Client support
+
+The low-level client provides the following methods concerned with user blocking.
```
Comment (laevandus) [inline docusaurus/docs/iOS/swiftui/swiftui-cookbook/blocking-users.md:22]: Or `… methods related to user …`

Comment (martinmitrevski) [reply docusaurus/docs/iOS/swiftui/swiftui-cookbook/blocking-users.md]: done

File: docusaurus/docs/iOS/swiftui/swiftui-cookbook/blocking-users.md [hunk 3]
```diff
@@ -0,0 +1,197 @@
+---
+title: Blocking Users
+description: How to block and unblock users
+---
+
+## Introduction
+
+Blocking users is an essential feature in a chat app because it enhances user safety and experience. It allows individuals to protect themselves from harassment, spam, and unwanted interactions. By giving users control over their interactions, it helps maintain privacy, reduces the risk of cyber bullying, and promotes a respectful community atmosphere.
```
Comment (laevandus) [inline docusaurus/docs/iOS/swiftui/swiftui-cookbook/blocking-users.md:8]: "online bullying" - I wonder if this could make vale happy

Comment (martinmitrevski) [reply docusaurus/docs/iOS/swiftui/swiftui-cookbook/blocking-users.md:8]: I'm trying now with splitting them, let's see if it succeeds.

Comment (martinmitrevski) [reply docusaurus/docs/iOS/swiftui/swiftui-cookbook/blocking-users.md:8]: It didn't, trying with online

**Review by laevandus — APPROVED** [review]

**Pull request merged**